lazy_tensor_core/lazy_tensor_core/csrc/init_python_bindings.cpp

```diff
@@ -413,6 +416,63 @@ void InitLtcModuleBindings(py::module m) {
           };
           return GetTensorsDump(tensors, coverter);
         });
+  m.def("_dynamic_size",
+        [](at::Tensor& self) {
+          LazyTensor self_lazy_tensor = bridge::GetLtcTensor(self);
+          return bridge::AtenFromLtcTensor(
+              self_lazy_tensor.CreateFrom(ir::MakeNode<ir::ops::DynamicSize>(
+                  self_lazy_tensor.GetIrValue())));
+        });
+  m.def("_add_dim",
```

Comment:
i didn't anticipate that add_dim was dynamic.  the 'dim' constant is the dynamic part, i guess?  but how c ome there isn't any special handling of the 'dim' argument in this API?